qemu/tests/qmp_command.py

```diff
@@ -258,6 +258,9 @@ def send_cmd(cmd):
             # Remove avocado machine type
             vm_machines = params["machine_type"].split(':', 1)[-1]
             expect_o = [{'alias': vm_machines}]
+        elif qmp_cmd == "query-vnc":
+            vnc_port =  vm.get_vnc_port()
+            expect_o = [{'service': str(vnc_port)}]
```

Comment:
Ok, you remove the checking for filed `enabled` and `host`.  Could you explain a bit more ? Thanks!

Reply:
According to your suggestion: 
First obtain the vnc port from the qemu command line, and then compare whether there are the same ports in the query result of the 'query-vnc' command . So I remove the checking for filed enabled and host

Reply:
My suggestion is to use the vnc port in qemu command line, instead of assume it always be 5900.
It is not conflict with checking `enabled` and `host`.

And my question above is about the reason to remove these 2 check point ?   Is it meaningless from product view?

Reply:
> My suggestion is to use the vnc port in qemu command line, instead of assume it always be 5900.
> It is not conflict with checking `enabled` and `host`.
> 
> And my question above is about the reason to remove these 2 check point ? Is it meaningless from product view?
I can't say it doesn't make sense. I'll add it.